app-i18n/translate-shell/translate-shell-0.9.6.9.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Online command-line translator"
+HOMEPAGE="https://www.soimort.org/translate-shell/"
+SRC_URI="https://github.com/soimort/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="public-domain"
```

Comment:
https://github.com/soimort/translate-shell/blob/develop/LICENSE

GitHub says it's Unlicense. Please compare the text to be sure.

Reply:
Good catch, thanks! Website listed it as public domain, but the repo license and waiver files are indeed matching Unlicense text. Fixed.